packages/dev/core/src/Gizmos/gizmo.ts

```diff
@@ -204,6 +204,8 @@ export class Gizmo implements IGizmo {
     private _tempMatrix1 = new Matrix();
     private _tempMatrix2 = new Matrix();
     private _rightHandtoLeftHandMatrix = Matrix.RotationY(Math.PI);
+    private _forward = Vector3.Forward();
+    private _back = Vector3.Backward();

```

Comment:
No need for those. You have `Vector3.LeftHandedForwardReadOnly` and the corresponding right-handed version.